fuzz/asn1.c

```diff
@@ -217,9 +217,11 @@ static ASN1_PCTX *pctx;
     if (type != NULL) { \
         int len2; \
         BIO *bio = BIO_new(BIO_s_null()); \
+        if (bio != NULL) { \
```

Comment:
There ought to be a blank line above.  I.e. [spaces] \ like the original.  Line 224 can be deleted since it's the blank.

Reply:
Thanks, I have submit a 'fixup! fixup!' to fix it.